packages/flutter_robusta_firebase_core/example/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:very_good_analysis/analysis_options.3.1.0.yaml
```

Comment:
Should use the latest version `4.0.0` and maybe you need to upgrade `very_good_cli` to generate up-to-date template.